docs/using/authenticating-to-github.md

```diff
@@ -0,0 +1,35 @@
+# Authenticating to GitHub
+
+## How to login to GitHub
+
+1. In Unity find the **GitHub** window. If you can't find it, you can switch to it by choosing **GitHub** from the **Window** menu.
+
+    <img src="images/github-menu-item.png" alt="GitHub menu item in the Window menu" width="500px"/>
+
+1. Click the **Sign in** button at the top right of the window.
+
+    <img src="images/github-sign-in-button.png" alt="GitHub menu item in the Window menu" width="500px"/>
+
+1. In the **Authenticate** dialog, enter your username or email and password
+
+   <img src="images/github-authenticate.png" alt="GitHub menu item in the Window menu" width="350px"/>
+
+      If your account requires Two Factor Authencation. You will be prompted for your auth code.
```

Comment:
Authentication is mispelled, and if the phrase starts with an If, then it should have a comma, not a period.

Reply:
🙈